crates/bevy_reflect/src/map.rs

```diff
@@ -49,9 +49,12 @@ pub trait Map: Reflect {
     /// If no value is associated with `key`, returns `None`.
     fn get_mut(&mut self, key: &dyn Reflect) -> Option<&mut dyn Reflect>;
 
-    /// Returns the key-value pair at `index` by reference, or `None` if out of bounds.
+    /// Returns the key-value pair at `index` by references, or `None` if out of bounds.
```

Comment:
Nit: I think the original was actually correct (could be wrong)
```suggestion
    /// Returns the key-value pair at `index` by reference, or `None` if out of bounds.
```